init.lua

```diff
@@ -60,16 +54,75 @@ local function re_run_failed_jobs(full_repo, run_id)
     log.info('Api response body is '..res.body)
```

Comment:
In the line above we should say 'workflow' and not 'job'. The API request is for restarting a workflow.

Reply:
fixed